pvlib/tests/test_clearsky.py

```diff
@@ -569,19 +579,19 @@ def test_detect_clearsky_iterations(detect_clearsky_data):
     alpha = 1.0448
     with pytest.warns(RuntimeWarning):
         clear_samples = clearsky.detect_clearsky(
-            expected['GHI'], cs['ghi']*alpha, cs.index, 10, max_iterations=1)
-    assert (clear_samples[:'2012-04-01 10:41:00'] == True).all()
-    assert (clear_samples['2012-04-01 10:42:00':] == False).all()
```

Comment:
De Morgan wonders if this should use `.any()` instead of `.all()`?

Reply:
Oops, this comment is about the line `    assert not clear_samples['2012-04-01 10:42:00':].all()  # expected False`

Reply:
You are correct, should be `.any()`